src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -56,7 +56,12 @@ const AppRouter = () => {
         <Route path="/account/profile" component={Profile} />
         <Route path="/account/questions" component={Questions} />
         <Route path="/account/code-of-conduct" component={CodeOfConduct} />
-        <Route path="/login" component={Login} />
+        <Route
+          path="/login/:venueId/:customToken"
+          component={LoginWithCustomToken}
+        />
+        {/* @debt The /login route doesn't work since we added non-defaulted props to the Login component */}
+        {/*<Route path="/login" component={Login} />*/}
```

Comment:
[note] Pretty sure this is tech debt anyway, not sure it ever really worked currently?